core/src/connection/pool.rs

```diff
@@ -50,7 +50,8 @@ use std::{
     task::Context,
     task::Poll,
 };
-use void::Void;
+
+use self::task::PendingConnectionCommand;
```

Comment:
nit, I would not `use ...` this directly and instead follow the same pattern as EstablishedConnectionCommand referencing it by the `task` namespace. E.g. `task::PendingConnectionCommand`